Formula/abi-compliance-checker.rb

```diff
@@ -3,6 +3,7 @@ class AbiComplianceChecker < Formula
   homepage "http://ispras.linuxbase.org/index.php/ABI_compliance_checker"
   url "https://github.com/lvc/abi-compliance-checker/archive/1.99.9.tar.gz"
   sha256 "52b0daca89fcda73cde126497c8015ca823417074ba02fcff68b7acf2f45e516"
+  revision 1
```

Comment:
This is still failing, even after the revision bump: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/1959/version=mavericks/console